inst/configure

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
 # TODO: Handle non-Autoconf configure scripts that might not support --host
-emconfigure ./configure.orig --host=wasm32-unknown-emscripten
+# Bash is needed for scripts without shebang
+emconfigure bash ./configure.orig --host=wasm32-unknown-emscripten
```

Comment:
Thinking about this some more, perhaps this would be better as:

```suggestion
emconfigure bash -c './configure.orig --host=wasm32-unknown-emscripten'
```

Just in case the original `configure` is a binary for some reason, rather than a shell script.

Out of interest, do you know if there a reason why some `configure` scripts don't have a shebag line? I had thought that POSIX requires it, but I guess not?

Reply:
I think it is only required if you want to be able to invoke them directly, but I think R assumes a posix compliant shell script that it can run in bash.

I write all my configure scripts manually and it was ever needed so I did not bother  🤷

Reply:
OK added your suggestion.